c/busybox-1.22.0/sleep_true-no-overflow_true-valid-deref.c

```diff
@@ -107,7 +107,7 @@ static void bb_error_msg_and_die(const char *s, ...)
 // file ./libbb-dump.i line 1
 static void bb_show_usage(void)
 {
-  ;
+  abort();
```

Comment:
Related to the discussions in some other PRs: should this be `__VERIFIER_assume(false)`?

Reply:
It would work for the moment, but would cause havoc when someone steps up to add termination analysis/labels to the benchmark?!

Reply:
I think 'abort' is best: (a) does not have a loop and (b) is a proper C call.